packages/config/lib/options.ts

```diff
@@ -560,6 +560,12 @@ export const breakingRootOptions: Array<BreakingOption> = [
     isWarning: false,
     testingTypes: ['e2e'],
   },
+  {
+    name: 'slowTestThreshold',
```

Comment:
When was it decided this should not be a root level config? This seems inconsistent to me; although CT and E2E have different defaults for `viewportWidth` and `viewportHeight`, we don't stop the user from configuring those as root values. Why is `slowTestThreshold` different? If I want to have a single value for both, I think I should be allowed to.

Reply:
Update: we decided it's going to non root level. I still find this a little weird, but I don't think it's incorrect.